common/include/pcl/pcl_macros.h

```diff
@@ -448,3 +448,30 @@ aligned_free(void* ptr)
 #else
   #define PCL_IF_CONSTEXPR(x) if (x)
 #endif
+
+// [[unlikely]] can be used on any conditional branch, but __builtin_expect is restricted to the evaluation point
+// This makes it quite difficult to create a single macro for switch and while/if
+/**
+ * @def PCL_CONDITION_UNLIKELY
+ * @brief marks a conditional branch as more likely than the other
```

Comment:
How about "Marks that a condition will probably evaluate to false, and gives the compiler a hint for optimization" instead (or similar)? Since the macro is called _unlikely_, it is a bit confusing if the doc says it marks one branch as more _likely_

Reply:
Good catch